pegasus/emails/2017_teacher_application_accept_partner.md

```diff
@@ -1,6 +1,8 @@
 ---
+<% deadline = Date.today + 7 %>
+<% deadline_string = deadline.strftime("%b %d, %Y") %>
```

Comment:
I'm just sad we're not in Rails land; otherwise I could do 

    deadline.strftime("%b #{deadline.day.ordinalize}, %Y")

And get "May 11th, 2017" instead of just boring ol' "May 11, 2017"